zbus/src/error.rs

```diff
@@ -125,6 +129,7 @@ impl fmt::Display for Error {
             Error::Address(e) => write!(f, "address error: {e}"),
             Error::ExcessData => write!(f, "excess data"),
             Error::InputOutput(e) => write!(f, "I/O error: {e}"),
+            Error::InputOutputAddress(e, address) => write!(f, "I/O error: {e} for {address}"),
```

Comment:
This will need to be update too:

```suggestion
            Error::InputOutputAddress(e, address) => write!(f, "failed to connect to `{address}`: {e}"),
```